basic/aws_lambda_sdk/aws_lambda_sdk.tf

```
@@ -0,0 +1,65 @@
+# If you have a lambda already Terraformed, replace the references to module.lambda_function
+# with references to your aws_lambda_function resource, and remove this module.
+module "lambda_function" {
+  source  = "terraform-aws-modules/lambda/aws"
+  version = "2.36.0"
+
+  function_name = "your_lambda"
+  description   = "A Lambda to be called from impl.py"
+  handler       = "handler.lambda_handler"
+  runtime       = "python3.8"
+
+  source_path = "${path.module}/lambda_src"
+}
+
+# The AWS IAM Resources that enable Sym to invoke your Lambda functions.
+module "lambda_connector" {
+  source  = "symopsio/lambda-connector/aws"
+  version = ">= 1.0.0"
+
+  environment       = "main"
+  lambda_arns       = [module.lambda_function.lambda_function_arn]
+  runtime_role_arns = [aws_iam_role.sym_runtime_connector_role.arn]
+}
+
+# The Integration your Strategy uses to invoke Lambdas.
+resource "sym_integration" "lambda_context" {
+  type = "permission_context"
+  name = "lambda-context-main"
+
+  external_id = module.lambda_connector.settings.account_id
+  settings    = module.lambda_connector.settings
+}
+
+############ Flow with Lambda ARN as a Flow Variable ##############
+
+resource "sym_flow" "this" {
+  name  = "aws-lambda-from-sdk"
+  label = "AWS Lambda SDK Example"
+
+  implementation = "${path.module}/impl.py"
+  environment_id = sym_environment.this.id
```

Hmmmmm since things are in different files now, do we want to add additional wayfinder comments in the examples?

e.g. here we could add
```suggestion
  # The sym_environment resource is defined in environment.tf
  environment_id = sym_environment.this.id
```